src/sagemaker/local/image.py

```diff
@@ -452,15 +467,23 @@ def _cleanup(self):
         pass
 
 
-class _HostingContainer(object):
-    def __init__(self, command, startup_delay=5):
+class _HostingContainer(Thread):
+    def __init__(self, command):
+        Thread.__init__(self)
         self.command = command
-        self.startup_delay = startup_delay
         self.process = None
 
-    def up(self):
-        self.process = Popen(self.command)
-        sleep(self.startup_delay)
+    def run(self):
```

Comment:
This is identical to the code in train, just with a different command. Suggest refactoring into a utility function.